TicTacToe/backend/TicTacToeBoard.cpp

```diff
@@ -29,37 +29,37 @@ GameState TicTacToeBoard::getGameState() const {
                 }
             }
         }
-        if (state == 1)
+        if (state == P1_WIN_STATE)
             return Win1;
-        else if (state == -1)
+        else if (state == P2_WIN_STATE)
             return Win2;
     }
     for (auto i = 0; i < BOARD_SIZE / 3; ++i) {
         auto state = checkLine(str[i], str[i + 3], str[i + 6]);
-        if (state == 1)
+        if (state == P1_WIN_STATE)
             return Win1;
-        else if (state == -1)
+        else if (state == P2_WIN_STATE)
             return Win2;
     }
-    if (numOccupied() < 9)
+    if (numOccupied() < BOARD_SIZE)
         return Continue;
     return Draw;
 }
 
 int TicTacToeBoard::numOccupied() const  {
     auto occ = 0;
     for (auto c: d.getDescription())
-        occ += (c != '.') ? 1 : 0;
+        occ += (c != EMPTY_SQUARE) ? 1 : 0;
     return occ;
 }
 
 int TicTacToeBoard::checkLine(char x, char y, char z) const {
-    if (x == y && y == z && x == 'x')
```

Comment:
I switched around the order of conditions here because I believe it will strictly shorten the amount of time spent checking this condition, if only by a little bit.